boefjes/plugins/kat_webpage_analysis/find_images_in_html/normalize.py

```diff
@@ -0,0 +1,32 @@
+from typing import Union, Iterator
+from urllib.parse import urljoin
+
+from bs4 import BeautifulSoup
+
+import validators
+
+from octopoes.models import OOI
+from octopoes.models.ooi.network import Network
+from octopoes.models.ooi.web import URL
+
+from boefjes.job_models import NormalizerMeta
+
+
+def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterator[OOI]:
+    soup = BeautifulSoup(raw, "html.parser")
+    images = set([img["src"] for img in soup.find_all("img") if hasattr(img, "src")])
```

Comment:
You can also use the css selector to find `img` tags with a `src` attribute: `soup.select("img[css]")`

This can also be achieved with the same`find_all` method: `soup.find_all("img", src=True)`